plugins/mediatek-scaler/fu-mediatek-scaler-device.c

```diff
@@ -611,43 +629,60 @@ fu_mediatek_scaler_device_get_isp_status(FuMediatekScalerDevice *self, GError **
 	st_res = fu_mediatek_scaler_device_ddc_read(self, st_req, error);
 	if (st_res == NULL)
 		return FALSE;
-	if (!fu_memread_uint8_safe(st_res->data, st_res->len, 2, &isp_status, error))
+
+	if (!fu_memread_uint8_safe(st_res->data, st_res->len, 2, isp_status, error))
+		return FALSE;
+
+	return TRUE;
+}
+
+static gboolean
+fu_mediatek_scaler_device_is_update_success_cb(FuDevice *device, gpointer user_data, GError **error)
+{
+	FuMediatekScalerDevice *self = FU_MEDIATEK_SCALER_DEVICE(device);
+	guint8 expected_status = 2;
```

Comment:
can you add a define or enum in the rs file for the `2` (and others) please?

Reply:
fixed, thanks.